sw/device/lib/testing/test_framework/ottf_isrs.S

```diff
@@ -0,0 +1,348 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/test_framework/ottf_macros.h"
+
+.extern pxCurrentTCB
+.extern ottf_exception_handler
+.extern ottf_software_isr
+.extern ottf_timer_isr
+.extern ottf_external_isr
+
+// -----------------------------------------------------------------------------
+
+/**
+ * Save MEPC to the stack.
```

Comment:
It looks like MEPC is saved to the stack outside of this function.

Reply:
Fixed.